File: src/main/java/com/realyoungk/sdi/dto/TelegramChatDto.java
package com.realyoungk.sdi.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public record TelegramChatDto(
        @JsonProperty("id") Long id
) {
}

File: src/main/kotlin/com/realyoungk/sdi/dto/TelegramChatDto.kt
package com.realyoungk.sdi.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
data class TelegramChatDto(
    @JsonProperty("id") val id: Long
)